integration-cli/docker_api_containers_test.go

```diff
@@ -370,6 +370,108 @@ func TestBuildApiDockerfilePath(t *testing.T) {
 	logDone("container REST API - check build w/bad Dockerfile path")
 }
 
+func TestBuildApiDockerFileRemote(t *testing.T) {
+	server, err := fakeStorage(map[string]string{
+		"testD": `FROM busybox
+RUN echo from testD
```

Comment:
@tiborvass I don't understand the question. You don't think I should check the output from the build?